src/operators/tensor/core.cairo

```diff
@@ -1106,10 +1106,78 @@ trait TensorTrait<T> {
     ///     let result = tensor.cosh();
     ///     return result;
     /// }
-    /// >>> [1,0.5403,3.7622]
+    /// >>> [1,1.5403,3.7622]
     /// ```
     ///
     fn cosh(self: @Tensor<T>) -> Tensor<FixedType>;
+    /// #tensor.sinh
+    ///
+    /// ```rust
+    ///     fn sinh(self: @Tensor<T>) -> Tensor<T>;
+    /// ```
+    ///
+    /// Computes the hyperbolic cosine of each element in the input tensor.
+    /// 
+    /// ## Args
+    ///
+    /// * `self`(`@Tensor<T>`) - The input tensor.
+    ///
+    ///
+    /// ## Returns
+    ///
+    /// A new `Tensor<T>` of the same shape as the input tensor with 
+    /// the hyperbolic sine of the values in the input tensor.
+    ///
+    /// ## Example
+    ///
+    /// ```rust
+    /// fn sinh_example() -> Tensor<FixedType> {
+    ///     // We instantiate a 3D Tensor here.
+    ///     // tensor = [[0,1,2]]
+    ///     // for each value val we calculate hyperbolic sin with the formula:
+    ///     // (exp(val)-exp(-val))/2
+    ///     let tensor = fp8x23_tensor_1x3_helper();
+    ///     let result = tensor.sinh();
+    ///     return result;
+    /// }
+    /// >>> [0, 1.1752, 3.6269]
+    /// ```
+    ///
+    fn sinh(self: @Tensor<T>) -> Tensor<FixedType>;
+    /// #tensor.sinh
```

Comment:
Here should be the docstring for `tanh`

Reply:
Ah I just forgot to remove the print() when I was debugging...I removed all print()'s and also removed the import for 'use debug::PrintTrait' in all of the tests that still had it.